src/main/java/org/triplea/client/ui/javafx/ToggleSwitch.java

```diff
@@ -32,7 +32,7 @@ public BooleanProperty switchedOnProperty() {
     final Rectangle background = new Rectangle(width, height);
     background.setArcWidth(height);
     background.setArcHeight(height);
-    background.setFill(Color.WHITE);
+    background.setFill(Color.web("#B8242B"));
```

Comment:
Is it possible to apply CSS to specialized controls like this one so we don't have to specify colors in code like this?

Reply:
That's the future Plan. My current knowlegde of JavaFX doesn't goes as far as knowing how to create Nodes that are completely skinnable using CSS.

Reply:
It has something to do with extending Control and implementing `getSkin`, but I still haven't the slightest idea on how to create an instance of Skin